### PR TITLE
Add support for Jellyseerr

### PR DESCRIPTION
**Pull request by WoxWik**
This jellyseerr service compliments the other ones nicely. Its a fork of overseerr meant to support jellyfin. 

Details: https://github.com/Fallenbagel/jellyseerr

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -80,6 +80,35 @@ services:
...
+      - TZ=${TIMEZONE}
+    ports:
+      - 5055:5055
+    volumes:
+      - ./jellyseerr:/app/config
+    restart: always
+    labels:
+      - traefik.enable=true
+      ## HTTP Routers
+      - traefik.http.routers.jellyseerr-rtr.rule=Host(`jellyseerr.${HOSTNAME}`)
```
Comment (AdrienPoupa) [inline docker-compose.yml:97]: In general, the pattern of this project is to have all the projects in subfolders, could you please change that? same applies for the `homepage.href` label

File: docker-compose.yml [hunk 2]
```diff
@@ -80,6 +80,35 @@ services:
...
       - homepage.widget.url=http://radarr:7878/radarr
       - homepage.widget.key=${RADARR_API_KEY}
+  jellyseerr:
+    image: fallenbagel/jellyseerr:latest
+    container_name: jellyseerr
+    environment:
+      - LOG_LEVEL=debug
+      - TZ=${TIMEZONE}
+    ports:
+      - 5055:5055
```
Comment (AdrienPoupa) [inline docker-compose.yml:90]: there is no need to publish the ports with Traefik usually

File: docker-compose.yml (hunk 1 above)
Comment (WoxWik) [reply docker-compose.yml:97]: Overseerr unfortunately doesn't support baseurls. I tried for some time trying to get it to work with stripurls middleware and it wouldn't budge.

More info:
https://docs.overseerr.dev/extending-overseerr/reverse-proxy
https://github.com/sct/overseerr/issues/274

In that github issue somebody got it to work doing some messy stuff by adding nginx in front, but not sure its worth it.

File: docker-compose.yml [hunk 3]
```diff
@@ -80,6 +80,35 @@ services:
...
+    volumes:
+      - ./jellyseerr:/app/config
+    restart: always
+    labels:
+      - traefik.enable=true
+      ## HTTP Routers
+      - traefik.http.routers.jellyseerr-rtr.rule=Host(`jellyseerr.${HOSTNAME}`)
+      - traefik.http.routers.jellyseerr-rtr.tls=true
```
[the remark this answers is not in the document]
Comment (WoxWik) [reply docker-compose.yml]: No reason, I was just copying documentation, but fixed now.

File: docker-compose.yml (hunk 2 above)
Comment (WoxWik) [reply docker-compose.yml]: Very cool

File: docker-compose.yml (hunk 1 above)
Comment (AdrienPoupa) [reply docker-compose.yml]: If it is not supported, then it should definitely be an optional service in my opinion, I'll have a look when I get a chance :) thanks for the changes!

**Comment by AdrienPoupa** [comment]
Thanks a lot for this! I am merging it to a new branch while I investigate if it is possible to make it run in a subfolder. I'll merge it to the main branch soon :)

**Pull request merged**